pom.xml

```diff
@@ -44,7 +44,7 @@
         <dependency>
             <groupId>org.openehealth.ipf.modules</groupId>
             <artifactId>ipf-modules-cda-mdht</artifactId>
-            <version>3.5.2</version>
```

Comment:
was this rolled forward for a reason?

Reply:
They changed the name of the Act to Allergy Concern Act in the spec but not in the code (still Allergy Problem Act). I thought the latest version (January 2019) would have hat change but did not. I left it since using the latest version is probably better.

Reply:
ok